### DISCUSSION TITLE
Multi-user systems and two onedrives for a user

### DISCUSSION THREAD

**Discussion by jhdore** [abraunegg/onedrive#2721] [Q&A] [answered]
Slightly related-but-not questions, which I can't figure out because I am probably confusing myself. My situation is this: 

I have a family of OneDrive users, who all log in to one computer, currently a Kubuntu 24.04 pc. How do I go about setting the users on it to access their own OneDrive accounts? 

One of the users has both a Microsoft Family OneDrive account, and also a work OneDrive account. How can this use make use of both accounts *at the same time*? i.e. map each onedrive account to a different local folder, and sync? 

I understand this may already have been covered somewhere and I've just not found it - please direct me accordingly if so!

Thanks
J

**Comment by abraunegg** [comment, marked as answer]
@jhdore 
> How do I go about setting the users on it to access their own OneDrive accounts?

Login as each user and setup each user - there is nothing special regarding this. Configure each user their own systemd service for background syncing. When you do this correctly:
* Each user will have their *own* data in their home directory under `~/OneDrive`
* Each user will have their *own* systemd service running to keep things in sync

> One of the users has both a Microsoft Family OneDrive account, and also a work OneDrive account. How can this use make use of both accounts at the same time? i.e. map each onedrive account to a different local folder, and sync?

First setup the 'Family' account for the user as per above.

To setup the second account, you need to create a second configuration for that user. Please read:
* https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#configuring-the-client-to-use-multiple-onedrive-accounts--configurations

> I have a family of OneDrive users, who all log in to one computer, currently a Kubuntu 24.04 pc.

Please also ensure that you install the client the *correct* way - do not use the Ubuntu packages. Please install by following *these* instructions: 
* https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md
